app/javascript/packages/document-capture/components/status-message.tsx

```diff
@@ -7,19 +7,13 @@ export const Status = {
   ERROR: 'ERROR',
   SUCCESS: 'SUCCESS',
 };
+interface StatusMessageProps {
+  status?: string;
```

Comment:
status wasn't nullable before?
```suggestion
  status: string;
```